lib/osf-components/addon/components/file-widget/template.hbs

```diff
@@ -0,0 +1,6 @@
+<div ...attributes>
+    <FileWidget::Manager @node={{@node}} as |widgetManager|>
+        <FileWidget::SelectedList @manager={{widgetManager}} />
```

Comment:
Consider changing the names of the `manager` param to `fileWidgetManager` or `widgetManager`.